src/Bigtable/Admin/V2/Gapic/BigtableInstanceAdminGapicClient.php

```diff
@@ -135,22 +160,40 @@ class BigtableInstanceAdminGapicClient
 
     private static $projectNameTemplate;
     private static $instanceNameTemplate;
+    private static $appProfileNameTemplate;
     private static $clusterNameTemplate;
     private static $locationNameTemplate;
     private static $pathTemplateMap;
-    private static $gapicVersion;
-    private static $gapicVersionLoaded = false;
-
-    protected $grpcCredentialsHelper;
-    protected $bigtableInstanceAdminStub;
-    private $scopes;
-    private $defaultCallSettings;
-    private $descriptors;
+
     private $operationsClient;
 
+    private static function getClientDefaults()
+    {
+        return [
+            'serviceName' => self::SERVICE_NAME,
+            'serviceAddress' => self::SERVICE_ADDRESS,
+            'port' => self::DEFAULT_SERVICE_PORT,
+            'scopes' => [
+                'https://www.googleapis.com/auth/bigtable.admin',
+                'https://www.googleapis.com/auth/bigtable.admin.cluster',
+                'https://www.googleapis.com/auth/bigtable.admin.instance',
+                'https://www.googleapis.com/auth/bigtable.admin.table',
+                'https://www.googleapis.com/auth/cloud-bigtable.admin',
+                'https://www.googleapis.com/auth/cloud-bigtable.admin.cluster',
+                'https://www.googleapis.com/auth/cloud-bigtable.admin.table',
+                'https://www.googleapis.com/auth/cloud-platform',
+                'https://www.googleapis.com/auth/cloud-platform.read-only',
+            ],
+            'clientConfigPath' => __DIR__.'/../resources/bigtable_instance_admin_client_config.json',
+            'restClientConfigPath' => __DIR__.'/../resources/bigtable_instance_admin_rest_client_config.php',
+            'descriptorsConfigPath' => __DIR__.'/../resources/bigtable_instance_admin_descriptor_config.php',
+            'versionFile' => __DIR__.'/../../VERSION',
+        ];
+    }
+
     private static function getProjectNameTemplate()
     {
-        if (self::$projectNameTemplate == null) {
+        if (null == self::$projectNameTemplate) {
```

Comment:
Why are these using yoda conditions now? Is this a new convention in gapic?

Reply:
I think php-cs-fixer did this